nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/createindex/CreateIndexFacetImpl.java

```diff
@@ -112,9 +116,15 @@ public void on(RepositoryCreatedEvent createdEvent) {
   }
 
   private void maybeInvalidateIndex(final AssetEvent event) {
-     if(matchesRepository(event) && isEventRelevant(event)) {
-       invalidateIndex();
-     }
+    Asset asset = event.getAsset();
+    String formatName = asset.format();
+    if (formatName.equals(HelmFormat.NAME)) {
```

Comment:
maybe better? HelmFormat.NAME.equals(formatName)

Reply:
Yes, you are absolutely right! Fixed